RestClient.cpp

```diff
@@ -45,23 +36,16 @@ bool RestClient::dhcp(){
 RestClient::RestClient(const char* _host, int _port, const char* _fingerprint){
     host = _host;
     port = _port;
-    ssl = 1;
+    ssl = true;
     fingerprint = _fingerprint;
-    num_headers = 0;
-    if (!contentType) {
-        contentType = "application/x-www-form-urlencoded";  // default
-    }
+    contentType = "application/x-www-form-urlencoded";
 }
 
-RestClient::RestClient(const char* _host, int _port, int _ssl) {
+RestClient::RestClient(const char* _host, int _port, bool _ssl) {
```

Comment:
I am not inclined to change the type of _ssl without good reason.  This will break many users who may upgrade